code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1331,30 +1331,42 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	id = "neurotoxin"
 	description = "A strong neurotoxin that puts the subject into a death-like state."
 	color = "#2E2E61" // rgb: 46, 46, 97
-	boozepwr = 0 //custom drunk effect
+	boozepwr = 50
 	quality = DRINK_VERYGOOD
 	taste_description = "a numbing sensation"
 	glass_icon_state = "neurotoxinglass"
 	glass_name = "Neurotoxin"
 	glass_desc = "A drink that is guaranteed to knock you silly."
 
+/datum/reagent/consumable/ethanol/neurotoxin/proc/pickt()
+	return (pick(TRAIT_PARALYSIS_L_ARM,TRAIT_PARALYSIS_R_ARM,TRAIT_PARALYSIS_R_LEG,TRAIT_PARALYSIS_L_LEG))
+
 /datum/reagent/consumable/ethanol/neurotoxin/on_mob_life(mob/living/carbon/M)
-	M.Paralyze(60, 1, 0)
 	M.dizziness +=2
-	switch(current_cycle)
-		if(15 to 45)
-			if(!M.slurring)
-				M.slurring = 1
-			M.slurring += 3
-		if(45 to 55)
-			if(prob(50))
-				M.confused = max(M.confused+3,0)
-		if(55 to 200)
-			M.set_drugginess(55)
-		if(200 to INFINITY)
-			M.adjustToxLoss(2, 0)
-	..()
+	M.adjustBrainLoss(1*REM, 150)
+	if(prob(20))
+		M.adjustStaminaLoss(10)
+		M.drop_all_held_items()
+		to_chat(M, "<span class='notice'>You dont feel your hands!</span>")
+	if(current_cycle > 5 && prob(15))
+		M.add_trait(pickt(), "neurotoxin")
+		M.adjustStaminaLoss(10)
+	if(current_cycle > 30)
+		M.adjustBrainLoss(4*REM)
+	if(current_cycle > 50 && prob(15))
```

Comment:
let's indent this, if this fails there's no need to also check if it's greater than 50.